integration-tests/runtime-tests/src/tests/tx/bondedFinance/txBondedFinanceTests.ts

```diff
@@ -34,54 +22,24 @@ export class TxBondedFinanceTests {
   /**
    * Tests tx.bondedFinance.offer successfully. SUDO Check!
    */
-  private static async txBondedFinanceOfferTest(wallet) {
+  private static txBondedFinanceOfferTest(wallet: IKeyringPair) {
     // ToDo: Find good parameter for reward.asset
     const requestParameters = {
       beneficiary: wallet.publicKey,
       asset: 1,
-      bondPrice: 1,
-      nbOfBonds: 10000000000000,
-      maturity: {Finite:{returnIn:50}},
-      reward: {asset: [170,141,183,460,469,231,731,687,303,715,884,105,728], amount: 10, maturity: 1}
+      bondPrice: 10000000000000, // pub const MIN_VESTED_TRANSFER: u32 = 1_000_000;
```

Comment:
If we make `MIN_VESTED_TRANSFER` an associated constant, it becomes gettable over the API, and we can then not rely on magic constants in the tests

Reply:
Updated!